src/main/minio.js

```diff
@@ -403,7 +403,6 @@ export class Client {
           this.logHTTP(reqOptions, null, e)
           cb(e)
         })
-      req.once('finish', () => pipe.removeAllListeners('error'));
```

Comment:
https://github.com/minio/minio-js/commit/11e25028c51804243ec59cfabef2a3d393ddd503

Do you remember why you added this? Because now error handlers never get called because we  end up removing error handlers because `finish` fires as soon as client is done sending the request.